ui/components/component-library/button-primary/button-primary.scss

```diff
@@ -36,7 +36,7 @@
       background-color: var(--color-error-alternative);
     }
 
-    &.mm-button--disabled:active {
+    &.mm-button-base--disabled:active {
```

Comment:
Still using child components class name here. Can this just be `&--disabled:hover {`?